paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -337,35 +293,25 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
     }
 
     fun handlePrimaryButtonClick(screenState: USBankAccountFormScreenState) {
-        _currentScreenState.value = _currentScreenState.value.updateInputs(
```

Comment:
How do you know this is no longer necessary?

Reply:
Same as above.